datasets/ecommerce/pre_pro.py

```diff
@@ -0,0 +1,12 @@
+
+
+
+
+import pandas as pd
+
+df = pd.read_csv('/Users/kevin/project_java/COOL/datasets/ecommerce/update-sample.csv')
```

Comment:
why do we need this `pre_pro.py` file?

Reply:
The original datasets is not ordered by DataTime, and our cool system will produce wrong results.
This scripts re-order the original datasets by DataTIme.